frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -116,13 +123,13 @@ export function LineGraph_({
     const [labelIndex, setLabelIndex] = useState<number | null>(null)
     const [holdLabelIndex, setHoldLabelIndex] = useState<number | null>(null)
     const [selectedDayLabel, setSelectedDayLabel] = useState<string | null>(null)
-    const { createAnnotation, updateDiffType, createGlobalAnnotation } = !inSharedMode
-        ? useActions(annotationsLogic({ insightNumericId }))
-        : { createAnnotation: noop, updateDiffType: noop, createGlobalAnnotation: noop }
 
-    const { annotationsList, annotationsLoading } = !inSharedMode
-        ? useValues(annotationsLogic({ insightNumericId }))
-        : { annotationsList: [], annotationsLoading: false }
+    // insightAnnotationsLogic must be bound using BindLogic
+    const { createAnnotation } = !inSharedMode ? useActions(insightAnnotationsLogic) : { createAnnotation: noop }
```

Comment:
Same comment as before. Not against it if we have to do that but it would be great if the comment could be more descriptive about why it has to be bound that way.